src/models/app_configuration.js

```diff
@@ -42,7 +41,7 @@ AppConfiguration.addLinkedApplication = function(appData, alias) {
     app_id: appData.id,
     deploy_url: appData.deployment.httpUrl || appData.deployment.url,
     name: appData.name,
-    alias: alias || unidecode(appData.name).replace(/[^a-zA-z0-9]+/gi, "-").toLowerCase()
+    alias: alias || appData.name
```

Comment:
can you slugify the app name in that case as well?

Reply:
I don't get what you mean.

The slugification of the alias happens directly in `Git.createRemote()`. When do you want to slufigy the name and what would it imply ?

Reply:
The alias is also used with the `--alias` flag. If we automatically generate an alias for the user (if they don't provide it), we make it console-friendly

Reply:
If the user specifies an alias when creating / linking the app, then we use it (we assume the user knows what they're doing)

Reply:
OK, so if no alias is provided through CLI, we should slugify the name and store the slugified version as an alias.

I think this means we should move the `slugify()` out of `Git.createRemote()` and back in `app_configuration.js`, WDYT?

Reply:
we still need to have it called in `Git.createRemote`, as users may pass invalid git remote names (anyway `slugify` is idempotent so it's not an issue)

Reply:
if the user wants to have a specific alias it's not an issue for CLI arguments, but for git remotes it's an issue